code-studio/src/js/initApp/project.js

```diff
@@ -132,6 +132,9 @@ var projects = module.exports = {
    * Whether this project uses Firebase for data storage.
    */
   useFirebase: function () {
+    if (!current) {
+      return;
+    }
```

Comment:
Should this get removed now that we shouldn't ever get here without a current user?